DataStructuresMemories/ViewControllers/WebPresenters/WKWebViewPresenter.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  WKWebViewController.swift
+//  DataStructuresMemories
+//
+//  Created by Oleksii Kolomiiets on 4/21/18.
+//  Copyright © 2018 Oleksii Kolomiiets. All rights reserved.
+//
+
+import UIKit
+import WebKit
+
+class WKWebViewPresenter: UIViewController, WKNavigationDelegate, PresenterProtocol {
+   
+    var webView: WKWebView!
+    
+    @IBOutlet weak var containerView: WKWebView!
+  
+    @IBAction func tocuhBackButton(_ sender: Any) {
+        self.dismiss(animated: true)
+    }
+    
+    var link: String?
```

Comment:
I would suggest to include `link` and `url` into `PresenterProtocol`.
You may provide default implementation for `url` in `PresenterProtocol` extension.